### DISCUSSION TITLE
Support incoming Accept-Encoding gzip, deflate, br to send gzipped responses back and Be able to handle gzipped responses from downstream services

### DISCUSSION THREAD

**Discussion by golfingal72** [apollographql/router#1251] [Q&A] [answered]
We have quite large payloads and implemented sending Accept-Encoding to downstream services and also want the gateway to be able to response with GZIPPED payloads to the originating caller.

**Comment by bnjjj** [comment, marked as answer]
This is in progress https://github.com/apollographql/router/pull/1229 you can follow the pull request

**Reply by bnjjj** [reply]
@golfingal72 It has been merged and will be available in the next release `0.9.5`

**Reply by golfingal72** [reply]
AWESOME SAUCE

**Reply by golfingal72** [reply]
working great... thanks much
